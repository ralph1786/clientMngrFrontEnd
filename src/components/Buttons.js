import React from "react";
import { Link } from "react-router-dom";
import "./Button.css";

const Buttons = () => {
  return (
    <div className="button">
      <Link to="/login">
        <button className="btn-hover color-1">Provider</button>
      </Link>
      <Link to="/">
        <button className="btn-hover color-1">Parent</button>
      </Link>
    </div>
  );
};

export default Buttons;
