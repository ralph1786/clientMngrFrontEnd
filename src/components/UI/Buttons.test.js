import React from "react";
import Buttons from "./Buttons";
import { Link } from "react-router-dom";
import { shallow } from "enzyme";

it("makes a snapshot of component", () => {
  const wrapper = shallow(<Buttons />);
  expect(wrapper).toMatchSnapshot();
});

it("should have two <Link/> components", () => {
  const wrapper = shallow(<Buttons />);
  expect(wrapper.find(Link)).toHaveLength(2);
});

it("expects first button to say Provider", () => {
  const wrapper = shallow(<Buttons />);
  expect(
    wrapper
      .find("button")
      .first()
      .text()
  ).toEqual("Provider");
});
