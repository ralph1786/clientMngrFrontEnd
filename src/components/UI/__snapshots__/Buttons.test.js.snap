// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`makes a snapshot of component 1`] = `
<div
  className="button"
>
  <Link
    replace={false}
    to="/login"
  >
    <button
      className="btn-hover color-1"
    >
      Provider
    </button>
  </Link>
  <Link
    replace={false}
    to="/parent_login"
  >
    <button
      className="btn-hover color-1"
    >
      Parent
    </button>
  </Link>
</div>
`;
